index.html

```diff
@@ -0,0 +1,156 @@
+      <div class="mobile-menu">
+        <li class="menuBtn">
+          <i class="fa-solid fa-bars fa-2x"></i>
+        </li>
+        <div class="menu hidden">
+          <ul>
+            <a href="./about.html">About</a>
+            <a href="#">Program</a>
+            <a href="#">Join us</a>
+            <a href="#">Sponsor</a>
+            <a href="#">Updates</a>
+          </ul>
+        </div>
+        <li class="close-menu hidden"><i class="fa-solid fa-x fa-2x"></i></li>
+      </div>
+      <div class="description">
+        <h3 class="desc-header">"Hello! body lovers"</h3>
+        <h1>
+          Making your body smoothier? <br />
+          here is the dream seminar.
+        </h1>
+        <p class="p-1">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Facere
+          dolorum impedit recusandae ullam alias
+        </p>
+        <h3 class="time">2023.04.25(Tuesday) - 28(Friday)</h3>
+        <p class="p-2 venue">@ GetPo central station, 5th street</p>
+      </div>
+    </header>
+    <div class="program">
+      <h2>Main program</h2>
+
+      <div class="cards">
+        <div class="card">
+          <img src="./img/lectur.png" alt="" class="card-img" />
+          <h2 class="card-title">Lecture</h2>
+          <p class="card-text">
+            Listen to the speakers from various countries about the messages of
+            sharing and opening.
+          </p>
+        </div>
+        <div class="card">
+          <img src="./img/chat.png" alt="" class="card-img" />
+          <h2 class="card-title">Forum</h2>
+          <p class="card-text">
+            Listen to the speakers from various countries about the messages of
+            sharing and opening.
+          </p>
+        </div>
+        <div class="card">
+          <img src="./img/work.png" alt="" class="card-img" />
+          <h2 class="card-title">Workshop</h2>
+          <p class="card-text">
+            Listen to the speakers from various countries about the messages of
+            sharing.
+          </p>
+        </div>
+        <div class="card">
+          <img src="./img/fitness.png" alt="" class="card-img" />
+          <h2 class="card-title">Fitness</h2>
+          <p class="card-text">
+            it's really passion and over coming to be part of fitness world and
+            amazing hobby to do.
+          </p>
+        </div>
+        <div class="card">
+          <img src="./img/health.png" alt="" class="card-img" />
+          <h2 class="card-title">Healthy</h2>
+          <p class="card-text">
+            Making exercise is the first medication to take today and it's first
+            vaccine to take for every diease.
+          </p>
+        </div>
+      </div>
+      <button class="join">Join BB seminar in 2023</button>
+      <a href="#" class="see-prog">SEE THE WHOLE PROGRAM</a>
+    </div>
+    <br /><br />
+    <div class="speakers-card">
+      <h2>Guest Coaches</h2>
+      <div class="speakers"></div>
+    </div>
```

Comment:
- [ ] You did great work @husush23 😃, however, you need some changes for this section, Kindly make sure you stuck the design as it is required 👍 

#### Original design
![Capture3](https://user-images.githubusercontent.com/57408419/234557328-3c13d9a5-67be-4e9a-b15a-d93f9c5f9597.PNG)

#### Your design

![Capture4](https://user-images.githubusercontent.com/57408419/234557683-97ac10b3-136d-4c7c-b86b-754c0c988096.PNG)